src/post/PostSingleModal.js

```diff
@@ -59,41 +57,64 @@ export default class PostSingleModal extends Component {
   }
 
   render() {
-    const { content, sidebarIsVisible, reblog, isReblogged, openCommentingDraft } = this.props;
+    const {
```

Comment:
I don't like this pattern. If variables are meant to be used only once or twice or are just meant to be passed to another component better to use the full form of `this.props.name` instead of a huge extraction which is actually making the size of the whole code in the file bigger than smaller.